### PR TITLE
[Instrumentation.Hangfire] Update CHANGELOG for 1.5.0-beta.1 release

### LINKED ISSUE #1250: Anticipated Release Date
Hello! 

First off, thanks for all the hard work! I noticed some unreleased changes in `OpenTelemetry.Instrumentation.Hangfire` that I'd like to integrate into my project.

Any idea when these might get released?

Cheers

### PR DESCRIPTION
**Pull request by skwasjer**
Update CHANGELOG for 1.5.0-beta.1 release

Fixes #1250

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: I would put here 1.0.0-beta.5.

Lets wait for other @open-telemetry/dotnet-contrib-maintainers

Comment (skwasjer) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: Thanks for quick review. It was not entirely clear what the convention is (first contrib here), so my reasoning was to follow the OTEL v1.5 version as I saw this in some other package releases like `OpenTelemetry.Instrumentation.Http` and `OpenTelemetry.Instrumentation.AspNetCore` about 2 weeks ago.

I'll wait for further feedback before updating!

Comment (skwasjer) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: I did make an oversight in the date though :( Jul->Jun

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: Both cases should work, last big chunk of releases is based on just rising -beta.x suffix: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1223

**Comment by Kielek** [comment]
Fixes: #1250

File: src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md
Comment (fred2u) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: I always prefer follow a convention :)
For me both are good but I have a preference for a versioning decoupled of dependencies.
Because in this case, we have two dependencies.

Comment (utpilla) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: I think it should be fine to follow what the main repo instrumentation libraries are doing and use `1.5.0-beta.1` here.

**Review by fred2u — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:7]:
```suggestion
Released 2023-Jun-23
```

**Review by Kielek — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md
Comment (skwasjer) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md]: God, I feel so stupid... lol

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md]: It was fine, when you have created PR :)

**Comment by Kielek** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/releases/tag/Instrumentation.Hangfire-1.5.0-beta.1
https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Hangfire/1.5.0-beta.1

**Pull request merged**
